presto-native-execution/presto_cpp/main/common/Exception.h

```diff
@@ -47,13 +47,17 @@ class VeloxToPrestoExceptionTranslator {
         kTranslateMap = {
             {velox::error_source::kErrorSourceRuntime,
              {{velox::error_code::kMemCapExceeded,
-               {0x00020000,
-                "GENERIC_INSUFFICIENT_RESOURCES",
+               {0x00020007,
+                "EXCEEDED_LOCAL_MEMORY_LIMIT",
                 protocol::ErrorType::INSUFFICIENT_RESOURCES}},
               {velox::error_code::kMemAborted,
```

Comment:
what kind of error is kMemAborted? Should it also be EXCEEDED_LOCAL_MEMORY_LIMIT?

Reply:
This is specific to Velox and will create a new code.